Sources/TimeStorage.swift

```diff
@@ -23,7 +23,7 @@ public enum TimeStoragePolicy {
 /// Handles saving and retrieving instances of `TimeFreeze` for quick retrieval
 public struct TimeStorage {
     private var userDefaults: UserDefaults
-    private let kDefaultsKey = "KronosStableTime"
+    private let kDefaultsKey = "KronosStableTimeStorageKey"
```

Comment:
Prefix with `Prex-` as this is defined in app level and we want to avoid potential name conflict.

Reply:
I have updated the code, you can review again, thanks!